client/app/scripts/liveblog-edit/timeline.js

```diff
@@ -197,12 +197,42 @@ define([
             }
         };
     }])
-    .directive('itemActions', ['api', 'notify', 'gettext', 'asset', 'itemsService', 'modal',
+    .directive('postActions', ['api', 'notify', 'gettext', 'asset', 'itemsService', 'modal',
         function(api, notify, gettext, asset, itemsService, modal) {
         return {
             restrict: 'E',
             replace: true,
-            templateUrl: 'scripts/liveblog-edit/views/timeline-item-actions.html',
+            templateUrl: 'scripts/liveblog-edit/views/timeline-post-actions.html',
+            link: function(scope, elem, attrs) {
+                scope.removePost = function() {
+                    var remItems = scope.post.groups[1].refs;
+                    confirm().then(function() {
+                        api.posts.remove(scope.post).then(function(message) {
+                            notify.pop();
+                            notify.info(gettext('Post removed'));
+                            scope.remove({post:scope.post});
+                            for (var i = 0, item; i < remItems.length; i ++) {
```

Comment:
I don't think this is the role of the client to remove every items, it should be the server role.
You should only remove the post here IMO